### ISSUE TITLE
[Bug] with the orders given to support ships. Version 310321

### ISSUE THREAD

**Issue by muerteFR** [daid/EmptyEpsilon#1399]
I saw problems with the orders given to support ships:
1. Once the ship leaves the base, you cannot indicate any waypoints to defend, they are missing in several scenarios. On others it appears and works. (cf picture)
2. The "Back" button is missing. (cf picture)
3. Sometimes giving the order to defend a station does not appear.

![bug assist2](https://user-images.githubusercontent.com/61491656/114512945-f1479000-9c39-11eb-9902-2333a8470d94.jpg)

**Comment by hemmond** [comment]
In which missions it definitely does not work? And in which missions it definitely works as expected?

**Comment by muerteFR** [comment]
I don't know anymore. I would have to retest. :-(

**Comment by elKuzco** [comment]
I can confirm this issues on linux for basically  all scenarii 

**Methods**  

1) From Game Master screen : Spawn Human Navy ship 
2) From relay : Create a waypoint 
3) From Relay : Open communications > Defend Waypoint 

**Results** 

Like the screen above : empty dialogue option

**Comment by aBlueShadow** [comment]
After trying older releases, this issue seems to have been introduced between EE-2020.11.23 and  EE-2021.03.16.

There has been some comm refactoring between the two releases, so I'd suspect some regression due to this.
@muerteFR I suggest to rename the topic and call a bug a bug :-)

**Comment by daid** [comment]
Isn't this the `_` used for both for loops and translation marker issue? Might have been solved already in master?

**Comment by hemmond** [comment]
Checked that on stock version 2021.03.31 in scenario Battlefield, this functionality does not work. 

As o this commit [
157a229](https://github.com/daid/EmptyEpsilon/commit/157a229ecd8271b2cda435d5ec90b5813b47cba1), this functionality still does not work (at least in Battlefield scenario). 

Steps to reproduce:

1. Start scenario Battlefield
2. Spawn player ship and take its Relay station
3. Unpause game. 
4. Place at least 1 waypoint (I placed exactly one)
5. Hail friendly ship and select "Defend a waypoint"
6. See that no waypoint is shown...
